django/db/migrations/loader.py

```diff
@@ -295,8 +295,8 @@ def check_consistent_history(self, connection):
                     raise InconsistentMigrationHistory(
                         "Migration {}.{} is applied before its dependency "
                         "{}.{} on database '{}'.".format(
-                            migration[0], migration[1], parent[0], parent[1],
-                            connection.alias,
+                            migration_key[0], migration_key[1], parent[0],
+                            parent[1], connection.alias,
```

Comment:
This could go on one line I think. 

(Would be less than 120 chars and splitting `parent[0]` from `parent[1]` is a little weird.)

Reply:
Ah, I thought the limit was 80 characters. I was wondering why there were so many offending lines.